apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/cohorts/index.vue

```diff
@@ -18,65 +23,132 @@ if (route.query?.page) {
-    columnType: "ONTOLOGY",
-    filterTable: "collectionEvents",
+    id: "populationAgeGroups",
+    config: {
+      label: "Population age groups",
+      type: "ONTOLOGY",
+      ontologyTableId: "AgeGroups",
+      ontologySchema: "CatalogueOntologies",
+      columnId: "ageGroups",
+      filterTable: "collectionEvents",
+    },
     conditions: [],
   },
   {
-    title: "Sample categories",
-    refTableId: "SampleCategories",
-    columnId: "sampleCategories",
-    columnType: "ONTOLOGY",
-    filterTable: "collectionEvents",
+    id: "sampleCategories",
+    config: {
+      label: "Sample categories",
+      type: "ONTOLOGY",
+      ontologyTableId: "SampleCategories",
+      ontologySchema: "CatalogueOntologies",
+      columnId: "sampleCategories",
+      filterTable: "collectionEvents",
+    },
     conditions: [],
   },
   {
-    title: "Cohort Types",
-    refTableId: "ResourceTypes",
-    columnId: "type",
-    columnType: "ONTOLOGY",
+    id: "cohortTypes",
+    config: {
+      label: "Cohort types",
+      type: "ONTOLOGY",
+      ontologyTableId: "CohortTypes",
+      ontologySchema: "CatalogueOntologies",
+      columnId: "type",
+      filterTable: "Cohorts",
+    },
     conditions: [],
   },
   {
-    title: "Design",
-    refTableId: "CohortDesigns",
-    columnId: "design",
-    columnType: "ONTOLOGY",
+    id: "cohortDesigns",
+    config: {
+      label: "Design",
+      type: "ONTOLOGY",
+      ontologyTableId: "CohortDesigns",
+      ontologySchema: "CatalogueOntologies",
+      columnId: "design",
+    },
     conditions: [],
   },
-]);
+];
+
+const filters = computed(() => {
+  const filters = [...pageFilterTemplate];
+
+  // if there are not query conditions just use the page defaults
+  if (!route.query?.conditions) {
+    return filters;
+  }
+
+  // get conditions from query
+  const conditions = conditionsFromPathQuery(route.query.conditions as string);
 
-let search = computed(() => {
-  // @ts-ignore
-  return filters.find((f) => f.columnType === "_SEARCH").search;
+  // apply the query conditions to the filter object
+  conditions.forEach((condition) => {
+    // find the filter object that matches the condition
+    const filter = filters.find((f) => f.id === condition.id);
+    if (!filter) {
+      console.error(`Filter with id ${condition.id} not found`);
+      return;
+    }
+
+    switch (filter.config.type) {
+      case "SEARCH":
+        const searchFilter = filter as IFilter;
+        searchFilter.search = condition.search;
+        break;
+      case "ONTOLOGY":
+      case "REF_ARRAY":
+        const ontologyFilter = filter as IConditionsFilter;
+        if (!ontologyFilter.conditions) {
+          ontologyFilter.conditions = [];
+        }
+        if (condition.conditions) {
+          ontologyFilter.conditions = condition.conditions;
+        }
+        break;
+      default:
+        //@ts-ignore
+        throw new Error(`Filter type ${filter.config.type} not supported`);
+    }
```

Comment:
extract this from cohorts page to helper function for reuse in all pages , mergeWithDefaults(conditions).